docs/docs/getting-started.md

```diff
@@ -31,13 +31,10 @@ npm i --save stainless-api/stl-api#prisma-0.0.1     # If you are using Prisma
 // ~/libs/stl.ts
 
 import { makeStl } from "stainless";
-import { makeNextPlugin } from "@stl-api/next";
 
 export type Context = {};
 
-const plugins = {
-  next: makeNextPlugin(),
-};
+const plugins = {};
```

Comment:
@rattrayalex all `makeNextPlugin()` did was add statics so there's no longer a need for it, so you want me to keep it as a stub for now, or get rid of the empty `plugins` in this example, or what?

Reply:
Yeah let's leave it as a stub I think, so that we can demo the `makeStl({plugins:[next]})` concept to make it "click" that you can pass things like this as plugins... I'm assuming that things like the next package will need a plugin at some point, but maybe that's wrong.

Reply:
I don't feel strongly though